### DISCUSSION TITLE
Additional notification filters

### DISCUSSION THREAD

**Discussion by dboslee** [community/community#12410] [Other Feature Feedback, Questions, & Ideas] [answered]
I have been wishing for a way to filter notifications by the status of an issue or pr. For example I might want to filter by `is:closed` and `is:open` similar to what you can do on the issues tab of a github repository.

Edit: This discussion has been merged with the duplicate https://github.com/github/feedback/discussions/15591. Please upvote and repsonse to that discussion if you would like to see this feature implemented.

**Comment by usmonster** [comment]
💯 this. Maybe it can be merged into #15591 (or vice versa)?

**Reply by dboslee** [reply]
Fine with me since that discussion has more details.

**Comment by dboslee** [comment, marked as answer]
This discussion has been merged with the duplicate https://github.com/github/feedback/discussions/15591. Please upvote and repsonse to that discussion if you would like to see this feature implemented.
